aws/resource_aws_network_acl.go

```diff
@@ -118,6 +125,12 @@ func resourceAwsNetworkAcl() *schema.Resource {
 						"action": {
 							Type:     schema.TypeString,
 							Required: true,
+							DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
This diff suppress is fairly common. Worth splitting it out into a core level function?